zyte_common_items/items.py

```diff
@@ -612,3 +613,102 @@ class BusinessPlace(Item):
 
     #: Data extraction process metadata.
     metadata: Optional[BusinessPlaceMetadata] = None
+
+
+@attrs.define(slots=True, kw_only=True)
+class RealEstateListing(Item):
```

Comment:
Hey! Sorry, I'm late :) I think we should name it RealEstate instead of RealEstateListing, because of the following reasons:

1) in Zyte API the data type is going to be named realEstate
2) There could be confusion on what Listing mean, or confusion between RealEstateListing and ReealEstateList (the data type which can be extracted from the pages with multiple real estate entries, similar to Product vs ProductList)
3) Other similar data types are named without ..Listing